handler/src/test/java/io/netty/handler/ssl/SniHandlerTest.java

```diff
@@ -229,4 +250,112 @@ protected void configurePipeline(ChannelHandlerContext ctx, String protocol) thr
             group.shutdownGracefully(0, 0, TimeUnit.MICROSECONDS);
         }
     }
+
+    @Test
+    public void testReplaceHandler() throws Exception {
+        final String sniHost = "sni.netty.io";
+        LocalAddress address = new LocalAddress("test");
```

Comment:
can you make this random/unique (e.g. UUID) to avoid conflicts with other tests that may also be running?